Mathlib/CategoryTheory/Sites/SheafOfTypes.lean

```diff
@@ -783,8 +783,92 @@ theorem isSheaf_bot : IsSheaf (⊥ : GrothendieckTopology C) P := fun X => by
   simp [isSheafFor_top_sieve]
 #align category_theory.presieve.is_sheaf_bot CategoryTheory.Presieve.isSheaf_bot
 
+/--
+For a presheaf of the form `yoneda.obj W`, a compatible family of Elements on a Sieve
+is the same as a co-cone. One direction is true for Presieves.
+Note: This is related to `CategoryTheory.Presheaf.conesEquivSieveCompatibleFamily`
```

Comment:
```suggestion
For a presheaf of the form `yoneda.obj W`, a compatible family of elements on a sieve
is the same as a co-cone. One direction is true for presieves.
Note: This is related to `CategoryTheory.Presheaf.conesEquivSieveCompatibleFamily`
```

Reply:
Can you be a little more precise with the docstring? Here you are associating a `Cocone` to any `FamilyOfElements ...`, and below the other direction, right? It is the compatibility that requires `Sieve`, not the construction?